routes/awsAccess.ts

```diff
@@ -0,0 +1,8 @@
+import express from "express"
+import { addUserToGroupController } from "../controllers/awsAccess";
+const router = express.Router();
+const { verifyDiscordBot } = require("../middlewares/authorizeBot");
+
+router.post("", addUserToGroupController);
```

Comment:
need to change this name addUserToGroupController

Reply:
updated this name